addons/payment_authorize/data/payment_acquirer_data.xml

```diff
@@ -6,7 +6,7 @@
         <field name="inline_form_view_id" ref="inline_form"/>
         <field name="support_authorization">True</field>
         <field name="support_fees_computation">False</field>
-        <field name="support_refund"></field>
+        <field name="support_refund">full_only</field>
```

Comment:
This'll need an Upgrade PR =]

Reply:
I have the upgrade PR ready, I will wait until the PR here is almost done and then push.